fastwsgi/common.c

```diff
@@ -44,3 +44,22 @@ int64_t get_env_int(const char * name)
     return v;
 }
 
+int64_t get_obj_attr_int(PyObject * obj, const char * name)
+{
+    PyObject * attr = PyObject_GetAttrString(obj, name);
+    Py_XDECREF(attr);
```

Comment:
Should `attr`'s ref count be decreased here? 
Shouldn't it be decreased after its used?

Reply:
This is done on purpose. The structure `g_srv` must not own this object (which is returned by the function `PyObject_GetAttrString`); completely feasts the contents of this object into itself.

When calling the command `del fastwsgi.server`, all attributes of the global object `fastwsgi.server` will also be destroyed, because they will have `refcnt` = 1